builtins-test-intrinsics/src/main.rs

```diff
@@ -682,7 +682,6 @@ pub fn _Unwind_Resume() {}
 
 #[cfg(not(any(windows, target_os = "cygwin")))]
 #[lang = "eh_personality"]
-#[unsafe(no_mangle)]
```

Comment:
Applied this in https://github.com/rust-lang/compiler-builtins/pull/899 since it's failing all PRs